pkg/component/monitoring/charts/bootstrap/templates/alertmanager/alertmanager.yaml

```diff
@@ -59,6 +59,8 @@ spec:
       labels:
         component: alertmanager
         role: monitoring
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
```

Comment:
Just a query actually, do we also need `networking.gardener.cloud/to-private-networks: allowed` here ? May be, somebody would like to setup their email server in a private network. I am not expert on how the network is setup in gardener. Feel free to ignore my comment if I am talking something absurd.

Reply:
> do we also need `networking.gardener.cloud/to-private-networks: allowed` here?

Hi @abdasgupta, I also considered that but I could not think of a use case so I opted for skipping granting access to private networks in general. The SMTP server configuration is done on the Gardener landscape level (so it is not configurable e.g. for shoot owners) and we are using an SMTP server with a public IP, so I thought that public network access is sufficient for now. I think even in other Gardener landscapes, landscape operators probably use an SMTP server with a public IP because in the current setup the same IP has to be reachable from all the seeds.

With this reasoning, granting access to private networks is not required and I'm a bit unsure about the risk of granting general private network access if it is not necessary, so I'd proceed like this and looking for advice from @acumino for a second opinion. :)

Reply:
Okay. I understood your reasoning @istvanballok . @acumino hope we are good. We can try to merge this PR then.

Reply:
@istvanballok Please add `networking.gardener.cloud/to-private-networks: allowed` as well. There is no way of knowing if the mail server is in the private or public network. There might as well be Gardener installations that use a private mail server.

Reply:
👍 I amended the commit and the PR description.